jooby/src/test/java/org/jooby/internal/RoutePatternTest.java

```diff
@@ -139,6 +139,72 @@ public void any2() {
   }
 
   @Test
+  public void anyNamed() {
+    new RoutePathAssert("GET", "com/**:rest")
+        .matches("GET/com/test.jsp", vars -> assertEquals("test.jsp", vars.get("rest")))
+        .matches("GET/com/a/test.jsp", vars -> assertEquals("a/test.jsp", vars.get("rest")))
+        .matches("GET/com/", vars -> assertEquals("", vars.get("rest")))
+        .butNot("GET/test.jsp");
```

Comment:
Is it worth testing `matches("GET/com", vars -> assertEquals("", vars.get("rest")))`? 
Notice that it doesn't has the last `/` in one of your test, it seems to me another corner case but not 100% sure.